glide-core/src/client/value_conversion.rs

```diff
@@ -50,34 +53,12 @@ pub(crate) fn convert_to_expected_type(
         },
         ExpectedReturnType::MapOfStringToDouble => match value {
```

Comment:
you can replace `MapOfStringToDouble` with `Map(&Some(BulkString), &Some(Double))` and delete it
Same with other `MapOf...` types we have